Documentation/ceph-object-bucket-claim.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim
+
+Rook supports the creation of new buckets (and access to existing buckets) via two custom resources:
```

Comment:
I know I suggested the above sentence but I now think it's better with out the parenthesis.